### ISSUE TITLE
README example not working as expected.

### ISSUE THREAD

**Issue by zoj613** [aesara-devs/aehmc#49]
The readme example throws an exception on the latest version. It requires a missing positional argument `step_size` in the `kernel` function call.

**Please provide a minimal, self-contained, and reproducible example.**

**Please provide the full traceback of any errors.**
```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-4acbdcf584da> in <module>
     16 # Build the transition kernel
     17 srng = RandomStream(seed=0)
---> 18 kernel = nuts.kernel(
     19     srng,
     20     logprob_fn,

TypeError: kernel() missing 1 required positional argument: 'step_size'
```

**Please provide any additional information below.**

## Versions and main components

* Aesara version: 2.3.3
* Python version: 3.10.0
* Operating system: Arch
* How did you install Aesara: pip

**Comment by zoj613** [comment]
The problem seems to be:
https://github.com/aesara-devs/aehmc/blob/abcc618f91fa03c3a26bdb425619617352bb2dff/aehmc/hmc.py#L27

That argument is missing from the example.

**Comment by rlouf** [comment]
Did you install `aehmc` via pip? If so can you try to see if the example works with the latest commit?

The kernel API has changed with [ab5125](https://github.com/aesara-devs/aehmc/commit/ab512549d58da8894cd64dbaadbae06a31a94fa3) and there hasn't been a release since.

**Comment by rlouf** [comment]
@zoj613 did you try to run the example with the latest commit to see if that's the issue?

**Comment by zoj613** [comment]
It works, I get `0.9300463205925347` as the printed output. i do get a 
```
WARNING (aesara.scan.basic): When the number of steps is fixed and equal to 1, the provided stopping condition, Elemwise{eq,no_inplace}.0 is ignored
WARNING (aesara.scan.basic): When the number of steps is fixed and equal to 1, the provided stopping condition, Elemwise{eq,no_inplace}.0 is ignored
```
warning but im not sure how important it is.

**Comment by rlouf** [comment]
Getting fixed in aesara-devs/aesara#741
